ingest/example/bulk_pretty

```diff
@@ -0,0 +1,27 @@
+{  
+  "index":{  
+    "_id":"1",
+    "_index":"dtdi",
+    "_type":"mineral-observation"
+  }
+}{  
+  "locality":[  
+    {  
+      "country":"Canada",
+      "region":"Québec, Canada",
+      "mine":"Poudrette quarry (Demix quarry; Uni-Mix quarry; Desourdy quarry; Carrière Mont Saint-Hilaire), Mont Saint-Hilaire, La Vallée-du-Richelieu RCM, Montérégie, Québec, Canada",
+      "coordinates":{  
+        "lat":"45.56277778",
+        "lon":"-73.14166667"
+      },
+      "mindat":"http://www.mindat.org/loc-599.html"
+    }
+  ],
+  "description":"Named after the Abenaki, the indigenous people who inhabited the area around Mont Saint-Hilaire in southern Québec, prior to the arrival of European settlers.",
+  "mindat":"http://www.mindat.org/min-2.html",
+  "mineral":{  
+    "name":"Abenakiite-(Ce)",
+    "rruffchemistry":"Na_26_Ce^3+^_6_(SiO_3_)_6_(PO_4_)_6_(CO_3_)_6_(S^4+^O_2_)O",
```

Comment:
So the format I adopted ("Na_26_Ce^3+^_6_....) is one of the standard syntaces, where _x_ means subscription and ^x^ means superscription. 
Another syntax is using html format, where <sub>x</sub> means subscription and <sup>x</sup> means superscription.
These two syntaces are obviously interchangeable. Maybe we can use the html style since we can easily display it in the browser.